safe/gui/tools/help_dialog.py

```diff
@@ -33,7 +33,9 @@ def __init__(self, parent=None, message=None):
         :type parent: QWidget
         """
 
-        QtGui.QDialog.__init__(self, parent)
+        QtGui.QDialog.__init__(
+            self, parent,
+            flags=Qt.WindowMinimizeButtonHint | Qt.WindowMaximizeButtonHint)
         self.setupUi(self)
         self.setWindowTitle(self.tr('InaSAFE %s Help' % get_version()))
```

Comment:
I think we should have convention on how we set title or text in the dialog. I prefer to do anything beside the static layout / widget in python code. Set text is in python code for me.